src/contexts/Api/defaults.ts

```diff
@@ -9,6 +9,26 @@ import {
   APIContextInterface,
   ConnectionStatus,
 } from 'contexts/Api/types';
+import { NetworkName } from 'types';
+
+const isNetworkName = (value: unknown): value is NetworkName =>
+  Object.values<unknown>(NetworkName).includes(value);
+
+const defaultNetworkName =
+  process.env.NODE_ENV === 'production' &&
+  process.env.REACT_APP_DISABLE_MAINNET !== '1'
+    ? NetworkName.AlephZero
+    : NetworkName.AlephZeroTestnet;
```

Comment:
Maybe a nitpick, but... we use the same env variables to add networks in the config. Instead of replicating it here, we could check if they exist in the config, and show the right one only based NODE_ENV.

Also, someone could set both REACT_APP_DISABLE_MAINNET and REACT_APP_DISABLE_TESTNET, but in this case there's not much we could do since there's no other network set and the app will crush either way...

Reply:
That's a good idea actually 👍

---

> someone could set...

Yeah, I noticed that loophole as well. I tried to make the _default network_ optional, but too much code depends on it not being non-optional so the PR would grow significantly and thus I backed out.

There's actually one network that's present regardless of any env vars: [the devnet](https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/blob/2bf481217460ad2ee7b578ce059da78316bc8b90/src/config/networks.ts#L218) - do you want me to use it as the _last resort fallback_?

Reply:
Ok, I did it that way to accelerate. If you want me to remove it after all - lmk.